source/iface/usm_allocations.rst

```diff
@@ -94,6 +94,199 @@ the ``device`` and ``context``.
 
 .. seealso:: `C++ Allocator <https://en.cppreference.com/w/cpp/named_req/Allocator>`__
 
+
+.. _allocation_functions :
+
+====================
+Allocation functions
+====================
+
+=======================
+``sycl::malloc_device``
```

Comment:
Maybe better to have hierarchy of sections (make sub-sub-section instead of sub-section for alloc functions):
```
-----------------------
``sycl::malloc_device``
-----------------------
```

Reply:
Thanks for hints...lets wait also review from Khronos and align